web/static/css/dfeed.css

```diff
@@ -1429,3 +1429,234 @@ body.frame .post-gravatar {
 	-webkit-border-radius: 5px;
 	border-radius: 5px;
 }
+
+a {
+	text-decoration-line: none;
+}
```

Comment:
No general rules. All added rules **must** be confined to the new view mode. This is a hard requirement for getting this change in.